include/small/interface_abstract.hpp

```diff
@@ -17,6 +17,8 @@
 #include <stdexcept>
 #include <type_traits>
 
+#define SMALL_HAS_FLOAT_SUPPORT 1
```

Comment:
Remove.  This should be defined in the platform files (like params.h) only if there is a FloatBuffer).

Reply:
Still in code base.